scripts/tests/test_zkai_snarkjs_statement_envelope_benchmark.py

```
@@ -0,0 +1,173 @@
+            BENCH.statement_commitment(envelope["statement"]),
+        )
+
+    def test_raw_proof_only_accepts_metadata_relabeling_with_fake_verifier(self) -> None:
+        _category, envelope = BENCH.mutated_envelopes()["model_id_relabeling"]
+
+        BENCH.verify_proof_only(envelope, external_verify=fake_external_verify)
+
+    def test_statement_envelope_rejects_model_id_relabeling(self) -> None:
+        _category, envelope = BENCH.mutated_envelopes()["model_id_relabeling"]
+
+        with self.assertRaisesRegex(BENCH.SnarkjsEnvelopeError, "statement policy mismatch"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)